tests/bsim/bluetooth/host/adv/chain/src/main.c

```diff
@@ -31,11 +31,19 @@
 		bs_trace_info_time(1, __VA_ARGS__);	\
 	} while (0)
 
-#define NAME_LEN 30
+#if defined(CONFIG_BT_DEVICE_NAME_DYNAMIC)
+#define NAME_LEN (CONFIG_BT_DEVICE_NAME_MAX + 1U)
+#else
+#define NAME_LEN sizeof(CONFIG_BT_DEVICE_NAME)
+#endif
```

Comment:
Ditto; this change can be reverted

Reply:
Ditto I will remove the conditional compile